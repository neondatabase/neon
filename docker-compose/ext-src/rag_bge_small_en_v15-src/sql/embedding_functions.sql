-- Embedding function tests
SELECT 'embedding_for_passage_test' AS test_name, 
       vector_dims(rag_bge_small_en_v15.embedding_for_passage('the cat sat on the mat')) > 0 AS result;

SELECT 'embedding_for_query_test' AS test_name,
       vector_dims(rag_bge_small_en_v15.embedding_for_query('the cat sat on the mat')) > 0 AS result;
