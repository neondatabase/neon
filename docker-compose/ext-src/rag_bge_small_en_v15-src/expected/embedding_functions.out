-- Embedding function tests
SELECT 'embedding_for_passage_test' AS test_name, 
       vector_dims(rag_bge_small_en_v15.embedding_for_passage('the cat sat on the mat')) > 0 AS result;
         test_name          | result 
----------------------------+--------
 embedding_for_passage_test | t
(1 row)

SELECT 'embedding_for_query_test' AS test_name,
       vector_dims(rag_bge_small_en_v15.embedding_for_query('the cat sat on the mat')) > 0 AS result;
        test_name         | result 
--------------------------+--------
 embedding_for_query_test | t
(1 row)

